zio-k8s-client/src/main/scala/com/coralogix/zio/k8s/client/impl/ResourceClient.scala

```diff
@@ -66,7 +66,7 @@ final class ResourceClient[
       }.map { initialResponse =>
         val rest = ZStream.fromPull {
           for {
-            nextContinueToken <- Ref.make(initialResponse.metadata.flatMap(_.continue)).toManaged
+            nextContinueToken <- ZIO.scoped(Ref.make(initialResponse.metadata.flatMap(_.continue)))
```

Comment:
No need for `ZIO.scoped` here

Reply:
Isn't `ZIO.scoped` the replacement for `.toManaged`?

Reply:
updated.